### PR TITLE
Mention smallterrain replaced by STK World Terrain in Changes.MD

### PR DESCRIPTION
**Pull request by abwood**
Also a minor doc fix.  The previous tileset link was broken (unversioned).

This is for #2593

### CODE REVIEW

**Comment by mramato** [comment]
Tahsnk @abwood. If `assets.agi.com` is what everyone should be using, you should probably do a global server in the code base for `cesiumjs.org/stk-terrain`, it's still used in quote a few places.

**Comment by abwood** [comment]
@mramato,
I went ahead and updated all of the links in the reference doc and updated the urls in the unit tests as well.  I think I got everything.

File: CHANGES.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Change Log
   * Removed `Camera.transform`, which was deprecated in Cesium 1.6. Use `Camera.lookAtTransform`.
   * Removed the `direction` and `up` options to `Camera.flyTo`, which were deprecated in Cesium 1.6. Use the `orientation` option.
   * Removed `Camera.flyToRectangle`, which was deprecated in Cesium 1.6. Use `Camera.flyTo`.
+* Deprecated
+  * The `smallterrain` tileset has been deprecated and will be removed in 1.11. Use the [STK World Terrain](http://cesiumjs.org/data-and-assets/terrain/stk-world-terrain.html) tileset instead.
```
Comment (emackey) [inline CHANGES.md]: This is a new link to `cesiumjs.org` when all of the other links in this PR have been changed to `assets.agi.com`.  Is this correct?

Comment (abwood) [inline CHANGES.md]: Yes, this link is correct.  The link is to the cesium website where we have the data-and-assets pages.

**Comment by pjcozzi** [comment]
Is this ready?

**Comment by abwood** [comment]
Yes

**Comment by pjcozzi** [comment]
Thanks @abwood

**Pull request merged**